module-4/backend/week_four.md

```diff
@@ -1,22 +1,64 @@
 ## Week Four - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
-* What's the purpose of a project management tool?
-* What are the stages of a card when using Pivotal Tracker?
-* What are the four types of cards available in Pivotal Tracker?
-* What's your favorite project management tool?
-* Why do we create staging environments?
-* What are the characteristics of a good README (in your opinion)?
-* What's one main improvement you're going to make to your code regarding accessibility issues?
-* What are some basic security concerns to be aware of when building applications?
-* Why is continuous integration helpful/important?
-* What are some continuous integration tools?
+1. What's your favorite project management tool?
+
+Personally, I really like Pivotal Tracker. Pivotal's granularity and features makes it more robust and a better overall tool than Waffle. I enjoy being able to break individual feature stories/bugs/chores into sub tasks, use labels, create milestones/epics and estimate time.
```

Comment:
nice! you could also check out some other tools.. Clubhouse, Trello, Asana to name a few!